eslintrc.json

```diff
@@ -8,53 +8,16 @@
     "jest": true,
     "node": true
   },
+  "plugins": [
+    "sort-requires"
+  ],
   "rules": {
-    "array-bracket-spacing": [ 2, "never" ],
-    "arrow-spacing": 2,
-    "block-spacing": 2,
-    "comma-dangle": [ 2, "always-multiline" ],
-    "comma-spacing": 2,
-    "comma-style": [ 2, "last" ],
-    "eol-last": 2,
```

Comment:
In dropping `eol-last` should `insert_final_newline = true` be dropped from all of stylelints [.editorconfig](https://github.com/stylelint/eslint-config-stylelint/blob/master/.editorconfig) files?